task-list/app/views/shared/_task_list.html.erb

```diff
@@ -0,0 +1,50 @@
+<table>
+  <thead>
+    <tr>
+      <th colspan="5">
+        <h1><%= @name %> Task List</h1>
+      </th>
+    </tr>
+    <tr>
+      <th></th>
+      <th>
+        Task
+      </th>
+      <th>
+        Completed Date
+      </th>
+      <th>
+        Who?
+      </th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @tasks.each do |task| %>
+    <tr>
+      <td class="bigger_checkbox">
```

Comment:
CSS class names with multiple words should be separated with the `-` not the `_`